### PR TITLE
add more profile links

### PR DESCRIPTION
**Pull request by matthewdias**

### CODE REVIEW

File: spec/factories/profile_link_sites.rb [hunk 1]
```diff
@@ -131,6 +131,60 @@
       validate_find '\A(https?://)?(www.)?(patreon.com/)?(?<username>[a-zA-Z0-9_\-]+)\z'
       validate_replace 'https://patreon.com/\k<username>'
     end
+
+    # DeviantArt
+    trait :deviantart do
+      validate_find '\A(https?://)?(www.)?(?<username>[a-zA-z0-9_\-]+)(.deviantart.com)?\z'
```
[the remark this answers is not in the document]
Comment (matthewdias) [reply spec/factories/profile_link_sites.rb:137]: hush

File: db/seeds/profile_link_sites.yml [hunk 2]
```diff
@@ -93,3 +93,48 @@ patreon:
   name: Patreon
   validate_find: '\A(https?://)?(www.)?(patreon.com/)?(?<username>[a-zA-z0-9_\-]+)\z'
   validate_replace: 'https://patreon.com/\k<username>'
+deviantart:
+  id: 20
+  name: DeviantArt
+  validate_find: '\A(https?://)?(www.)?(?<username>[a-zA-z0-9_\-]+)(.deviantart.com)?\z'
```
Comment (NuckChorris) [inline db/seeds/profile_link_sites.yml]: I don't think underscores are valid and also can you do a www. Prefix?

**Review by NuckChorris — APPROVED** [review]
Looks solid!

File: db/seeds/profile_link_sites.yml (hunk 2 above)
Comment (Uricorn) [reply db/seeds/profile_link_sites.yml]: Underscores are indeed not valid.

File: spec/factories/profile_link_sites.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (toyhammered) [reply spec/factories/profile_link_sites.rb:137]: You can just throw a rubocop disable/enable metrics to all of these, we don't care if they go over the line limit.

File: db/seeds/profile_link_sites.yml (hunk 2 above)
Comment (wopian) [reply db/seeds/profile_link_sites.yml]: `www` isn't valid for profiles

Comment (matthewdias) [reply db/seeds/profile_link_sites.yml]: fixed

**Pull request merged**